models/Asset.php

```diff
@@ -2036,39 +2026,34 @@ private function updateChildPaths(FilesystemOperator $storage, string $oldPath)
      */
     private function relocateThumbnails(string $oldPath)
     {
-        $oldParent = dirname($oldPath);
-        $newParent = dirname($this->getRealFullPath());
+        $oldParent = dirname($oldPath).'/'.$this->getId();
+        $newParent = dirname($this->getRealFullPath()).'/'.$this->getId();
         $storage = Storage::get('thumbnail');
 
         try {
-            //remove source parent folder thumbnails
-            $contents = $storage->listContents($oldParent)->filter(fn (StorageAttributes $attributes) => ($attributes->isFile() && strstr($attributes['path'], 'image-thumb_')));
-            /** @var StorageAttributes $item */
-            foreach ($contents as $item) {
-                $storage->delete($item['path']);
+            //remove source parent folder preview thumbnails
+            $sourceFolder = Asset::getByPath($oldPath);
```

Comment:
```suggestion
            $sourceFolder = Asset::getByPath(dirname($oldPath));
```
We need the parent folder of the current here, because `$oldPath` is not available anymore (it's `$this`, so just got moved 😉).